docs/content/concepts/automation/declarative-automation.mdx

```diff
@@ -0,0 +1,185 @@
+---
+title: "Declarative Automation | Dagster Docs"
+description: "Dagster can automatically materialize assets when criteria are met, enabling a declarative approach to asset materialization."
+---
+
+# Declarative Automation
+
+<Note>
+  This feature is currently <strong>experimental</strong>.
+</Note>
+
+Dagster can automatically materialize assets when criteria are met, enabling a declarative approach to asset materialization. Instead of defining explicit workflows to materialize assets, you describe the conditions under which they should be materialized and let the system kick off runs in response.
+
+For example, you have an asset that's scheduled to execute every day at midnight. Instead of running whether there's new data or not, you can use Declarative Automation to materialize the asset only when its parent has been updated.
```

Comment:
minor nit: maybe "running whether" -> "executing whether", and "only when its parent" -> "only after its parent". don't have strong reasoning for either of these, other than that it feels a little more direct to my ear so feel free to ignore